src/App.js

```diff
@@ -1344,10 +1415,20 @@ render() {
 
                   <Balance icon={xdai} selected={selected} text={"xDai"} amount={this.state.xdaiBalance} address={account} dollarDisplay={dollarDisplay}/>
                   <Ruler/>
+
```

Comment:
These line breaks could lead to future merge conflicts. I'd be happy if they were removed.